Description: Fix lintian's hyphen-used-as-minus-sign in manpages
 for esmtp(1) and esmtprc(5).
Origin: vendor
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>

--- a/esmtp.1
+++ b/esmtp.1
@@ -21,11 +21,11 @@
 
 \fBmailq\fR
 .br
-\fBsendmail -bp\fR
+\fBsendmail \-bp\fR
 
 \fBnewaliases\fR
 .br
-\fBsendmail -bi\fR
+\fBsendmail \-bi\fR
 
 .SH DESCRIPTION
 \fBEsmtp\fR is a send-only \fBsendmail\fR emulator for machines which normally
--- a/esmtprc.5
+++ b/esmtprc.5
@@ -87,7 +87,7 @@
 
 .TP
 \fBforce reverse_path\fR
-Set the envelope from address. The address given to -f will only
+Set the envelope from address. The address given to \-f will only
 be used as "From:" when the message contains none. "%u" will
 be replaced with the username. "%%" by "%".
 
@@ -146,7 +146,7 @@
 place a %T.  The mail message's \fBFrom\fR address will be inserted where you
 place an %F.
 
-Some common MDAs are "/usr/bin/procmail -d %T", "/usr/bin/deliver" and
+Some common MDAs are "/usr/bin/procmail \-d %T", "/usr/bin/deliver" and
 "/usr/lib/mail.local %T".
 
 .TP
@@ -160,7 +160,7 @@
 For example:
 
 .nf
-    mda = "procmail -d %T"
+    mda = "procmail \-d %T"
     force_mda = "\fIsomeuser\fR"
 .fi
 
